python/py-pybonjour/Portfile

```diff
@@ -1,9 +1,11 @@
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           gitlab 1.0
 
+gitlab.instance     https://dev.gajim.org
+gitlab.setup        lovetox pybonjour-python3 e7c010db6372559b9ee9cb64dde75206db684d86
```

Comment:
this should not take a hash, but a proper version number/tag

Reply:
They don’t have @1.1.3 in tags.